README.md

```diff
@@ -113,6 +115,37 @@ Output: 0,1,NORTH
 Simulation ended.
 ```
 
+You can also run the simulator and type in your commands one line at a time. Invoke the `stdin` rake task:
+
+```
+$ cd ~/toyrobot
+$ bundle exec rake stdin
```

Comment:
Do we really need to include the `$` shell prompt for each command-line example? And also the `cd ~/toyrobot`?